packages/react-components/src/components/Survey/TagsSurvey/TagsSurvey.tsx

```diff
@@ -204,11 +211,12 @@ export const _TagsSurvey = (props: _TagsSurveyProps): JSX.Element => {
       </Stack>
 
       <Pivot>
-        {Object.keys(tags).map((key, i) => {
+        {Object.keys(tags).map((k, i) => {
+          const key = k as keyof SurveyIssuesHeadingStrings;
```

Comment:
this is annoying object.keys doesn't auto type (apparently [for good reason]()), can we add this helper function
```ts
function getKeys<T extends object>(obj: T): (keyof T)[] {
  return Object.keys(obj) as Array<keyof T>;
}
```
and use that instead of Object.keys to auto type the `key` value and avoid this runtime extra line?